resources/themes/Night Mapping/palette.txt

```diff
@@ -1,7 +1,7 @@
 0:#535353
 2:#323232
 5:#535353
-6:#111111
+6:#aaaaaa
```

Comment:
@YoannQDQ , are you sure about this change? Have you checked a big chunk of the QGIS UI after this change to make sure no visual regression occurred (e.g. menu/symbol button's menu, style manager, option dialogs, etc.).

Reply:
Yeah, tested ;) ... is what I'd like to say.
The problem is that the colors are defined in three separate places:
- styles.qss
- variables.qss
- palette.txt

And widgets may use any combination of those colors.
So its rather difficult to anticipate the impact of a given palette change. Most colors in the palette seem largely unused, but the `QPalette::Text` color (6) IS used, by QLineEdit making them nearly illegible. I could not find any use of `QPalette::WindowText` though, so I think I'll just  use that as the MapTip text color.

Reply:
@YoannQDQ , right, please revert that change then please :) if you blame check the palette.txt, you'll see the value was changed from #aaaaaa to #111111 six months ago to fix another issue (see https://github.com/qgis/QGIS/commit/a5e65be03411778ca0d56481afbae77fedf61a0c).